ipalib/util.py

```diff
@@ -1029,6 +1030,31 @@ def has_managed_topology(api):
     return domainlevel > DOMAIN_LEVEL_0
 
 
+def print_replication_status(entry, verbose):
+    """Pretty print nsds5replicalastinitstatus, nsds5replicalastinitend,
+    nsds5replicalastupdatestatus, nsds5replicalastupdateend for a
+    replication agreement.
+    """
+
+    if verbose:
```

Comment:
It probably makes more sense to move the verbose conditional outside the function. Why call it at all if verbose is False?

Reply:
I guess it does defer the decision of print to the function and makes the calling code simpler. I withdraw my objection.

Is there a way we can test this? Maybe artificially add a replication agreement that we know will fail so won't have these attributes?

Reply:
It happened because of a lack of disk space at the "right" time during replication.
I'm not sure testing for that is going to be that easy...